_data/cloud.yml

```diff
@@ -46,6 +46,16 @@ websites:
           text: "Currently in beta as of June 5th 2014. Create a support ticket to request 2FA added to your organization"
       doc: https://support.cloud.engineyard.com/entries/40538256-Two-factor-authentication
 
+
+    - name: fortrabbit
+      url: https://www.fortrabbit.com/
```

Comment:
invalid ssl/tsl cert. Either fix the certificate issue or change the url to http. (Though https is preferred)